### ISSUE TITLE
Cache issue / laraval forge?

### ISSUE THREAD

**Issue by DrTrills** [statamic/cms#2800]
## Bug Description
My local environment (valet) would show any changes made instantly as expected. After the most recent update it seems cache on my forge server isn't working correctly. I have cache disabled on it as I am making many pushes to the server. This was working as expected. new push, instantly visible. 

As of today any changes pushed we'rent reflected live on the site. Clearing cache manually, using all the laravel and statamic commands and even SSH'd into the forge box to delete anything in the cache folders. Nothing worked. 

I just was able to get it working by enabling cache (full) and re-deploying the site. From there I went to each page 1 by 1 and reloaded it. It seems the cache rebuilt or overwrote what it previously had saved and is working now?

I am not sure if this is a Laravel or Statamic issue. Certainly it's related to cache. Also worth noting of the pages updated, a few did get the updates visible e.g. globally changed navbar which showed on some pages and not others) it also showed all the correct changes in the live preview area.

## How to Reproduce
Push changes to server, keep cache to null and it will get stuck. 

Enable cache (I did full, not sure about half) and reload the pages. It takes a second but they rebuild. 
See above.

## Environment

Statamic 3.0.23 Pro
Laravel 7.29.3
PHP 7.4.8
No addons installed

**Install method (choose one):**
- Fresh install from `statamic/statamic`

**Comment by damcclean** [comment]
Would you mind sharing what your Forge deploy script looks like?

**Comment by DrTrills** [comment]
```
cd /home/forge/beta.sitURL.com

#use the following if errors occur again.
#git fetch --all
#git reset --hard origin/main

#git stash

git pull origin main
$FORGE_COMPOSER install --no-interaction --prefer-dist --optimize-autoloader

( flock -w 10 9 || exit 1
    echo 'Restarting FPM...'; sudo -S service $FORGE_PHP_FPM reload ) 9>/tmp/fpmlock

yarn run dev

php artisan optimize
php please cache:clear
php please static:clear
php please stache:clear
php please glide:clear
```

**Comment by damcclean** [comment]
You should be able to remove almost all of the artisan and please commands at the end of your script.

If you want, you can keep php artisan cache:clear (which will clear both Laravel's cache and Statamic's stache.)

**Comment by jasonvarga** [comment]
That was a little hard to follow, but if you change your static caching setting to null, and try to run static:clear, it won't do anything.

If you've previously had full static caching enabled and have any pages generated, they'll still continue to be served even if you run static:clear (because again, if you disabled it, it the 'clear' command wouldn't clear anything).

Sounds like things might be working normally.

**Comment by DrTrills** [comment]
Thanks Jason, yeah apologies it was a bit of a ramble. Any suggestions on how I can safely disable cache for the time being? I suppose keep it full, clear it. then before hitting a page switch to null and save environment so it won't rebuild?
